apps/contracts/contracts/EcoEarn.sol

```diff
@@ -26,14 +26,24 @@
 pragma solidity ^0.8.19;
 
 import '@openzeppelin/contracts/access/AccessControl.sol';
-import './interfaces/IToken.sol';
+import './interfaces/IX2EarnRewardsPool.sol';
+import '@openzeppelin/contracts/utils/Strings.sol';
 
 /**
  * @title EcoEarn Contract
  * @dev This contract manages a reward system based on cycles. Participants can make valid submissions to earn rewards.
+ * Rewards are being distributed by interacting with the VeBetterDAO's X2EarnRewardsPool contract.
+ *
+ * @notice To distribute rewards this contract necesitates of a valid APP_ID provided by VeBetterDAO. This contract
+ * can be initially deployed without this information and DEFAULT_ADMIN_ROLE can update it later through {EcoEarn-setAppId}.
+ * This contract must me set as a `rewardDistributor` inside the X2EarnApps contract to be able to send rewards to users and withdraw.
```

Comment:
`be` typo

Reply:
Yes, in testenet devs should interact with dev environment. Will add to docs.

Reply:
Done